### ISSUE TITLE
Extend the Tagging Wizard with the possibility to change the file encodings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [octavian-paraschiv/protone-suite#211]
```
Extend the Tagging Wizard with the possibility to change the file encodings.

Current allowed changes:

Source WAV to Dest MP3 (Customizable MP3 params for Dest)

MP3 to WAV (WAV follows MP3's number of channels. Use 16-bit samples)

Source MP3 to Dest MP3 (Customizable MP3 params for Dest)
```

Original issue reported on code.google.com by `octavian...@gmail.com` on 20 Mar 2015 at 11:46

**Comment by octavian-paraschiv** [comment]
Implemented in build 2.1.4

**Comment by octavian-paraschiv** [comment]
Verified with build 2.1.24
